### ISSUE TITLE
Unlocking WiFi settings

### ISSUE THREAD

**Issue by erppaul** [kylejericson/JAMF#2]
Hi @kylejericson , 

I'm inquiring about the possibility of unlocking WiFi settings or enabling a low data mode for WiFi, similar to the command "security authorizationdb write system.preferences.network allow." I came across this GitHub repository: ttps://github.com/kylejericson/JAMF/blob/master/Scripts/Allow%20standard%20users%20rights%20to%20system%20prefs.sh, but it seems that the mentioned command isn't effective.

Best regards,
Paul

**Comment by kylejericson** [comment]
Paul did you try to run this command as a root user? I built this script to be run as root via Outset.

**Comment by erppaul** [comment]
Yes I have run it as root. An admin query still pops up when I try to make changes to the low data mode.
